### PR TITLE
Start cluster in Terraform.

### PR DESCRIPTION
**Pull request by jeschkies**
## High-level description

This change introduce a `make` targets to launch a hybrid cluster and run all `--windows-only`
tests.

## Corresponding DC/OS tickets (required)
- [D2IQ-65476](https://jira.d2iq.com/browse/D2IQ-65476)

### CODE REVIEW

**Comment by jeschkies** [comment]
@mesosphere-mergebot label Ready for Review

**Comment by jeschkies** [comment]
@timcharper, would be nice if you could review the Makefile.

**Comment by DominikDary** [comment]
While destroying this is failing:
[Step 3/3] ${var.ssh_public_key_file == "" ? var.ssh_public_key : file(local.ssh_public_key_file)}

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @janisz @DominikDary

File: test_util/Makefile [hunk 1]
```diff
@@ -1,70 +1,86 @@
...
-output:
-	./terraform output > ${OUTPUT_FILE}
+.PHONY: ssh 
+ssh: cluster.json
+	ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i $(SSH_KEY) centos@$(shell jq '.masters_public_ip.value' $<)
 
 .PHONY: destroy
 destroy:
-	./terraform destroy
+	$(TERRAFORM) destroy -auto-approve;
```
Comment (DominikDary) [inline test_util/Makefile]: Due to the error: 
`[09:41:56][Step 3/3] ${var.ssh_public_key_file == "" ? var.ssh_public_key : file(local.ssh_public_key_file)}`

don't we need to add here as well the public key like:
$(TERRAFORM) destroy -auto-approve ssh_public_key_file="$(SSH_KEY).pub";

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required Change to the automated testing

**Pull request merged**